aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/status/DBInstanceStatus.java

```diff
@@ -1,16 +1,17 @@
-package software.amazon.rds.dbinstance;
+package software.amazon.rds.dbinstance.status;
 
 import software.amazon.awssdk.utils.StringUtils;
 
-public enum DBInstanceStatus {
+public enum DBInstanceStatus implements TerminableStatus {
     Available("available"),
     Creating("creating"),
     Deleting("deleting"),
     Failed("failed", true),
     IncompatibleRestore("incompatible-restore", true),
     IncompatibleNetwork("incompatible-network", true),
     IncompatibleParameters("incompatible-parameters", true),
-    InaccessibleEncryptionCredentials("inaccessible-encryption-credentials", true);
+    InaccessibleEncryptionCredentials("inaccessible-encryption-credentials", true),
+    StorageFull("storage-full", true);
```

Comment:
Did you test storage-full recovery with this code? My concern if describe during storage-full modify call returns a stale status, the whole execution fails.

Reply:
I see your point. `isDBInstanceStabilizedAfterMutate` relies on this status as a non-terminal. We'd better get rid of this flag here. Thanks.